packages/pieces/community/text-helper/src/lib/actions/strip-html.ts

```diff
@@ -0,0 +1,17 @@
+import { stripHtml } from 'string-strip-html';
+import { createAction, Property } from '@activepieces/pieces-framework';
+
+export const stripHtmlContent = createAction({
+  name: 'stripHtml',
+  displayName: 'Strip HTML',
+  description: 'Strip HTML tags from a string',
```

Comment:
Could you please update the displayName to "Remove HTML Tags" and the description to "Removes every HTML tag and returns plain text"?